runtime/metricsview/executor.go

```diff
@@ -214,6 +215,10 @@ func (e *Executor) Query(ctx context.Context, qry *Query, executionTime *time.Ti
 			return nil, err
 		}
 
+		for k, v := range ast.olapContext {
+			args = append(args, sql2.NamedArg{Name: drivers.DialectDruid.ContextKeyArgPrefix() + k, Value: v})
```

Comment:
Could we avoid a `Druid` dialect hard-coding here by having something like `additionalArgs` instead of `olapContext`? 

Then when processing the spine in `ast.go`, the dialect might either return extra args or a range SELECT query.